Python/tests/test_runner_datetime.py

```diff
@@ -81,18 +81,26 @@ def test_datetime_model(culture, model, options, context, source, expected_resul
             assert_resolution(actual_resilution_value, expected_resoulution_value)
 
 def get_assert_model_resolution(option):
-    assert_resolutions = {
-        None: assert_model_resolution_option_none,
-        'CalendarMode': assert_model_resolution_option_calendar_mode,
-        'ExtendedTypes': assert_model_resolution_option_extended_types,
-        'SplitDateAndTime': assert_model_resolution_option_split_date_and_time
-    }
-    return assert_resolutions.get(option)
+    if option == 'CalendarMode':
+        return assert_model_resolution_option_calendar_mode
+    elif option == 'ExtendedTypes':
+        return assert_model_resolution_option_extended_types
+    elif option == 'SplitDateAndTime':
+        return assert_model_resolution_option_split_date_and_time
+    return assert_model_resolution_option_none
+
+def single_assert(actual, expected, prop):
+    if expected.get(prop):
+        assert actual[prop] == expected[prop]
+    else:
+        assert actual.get(prop) is None
 
 def assert_model_resolution_option_none(actual, expected):
-    assert actual.timex == expected['timex']
-    assert actual.type == expected['type']
-    assert actual.value == expected['value']
+    single_assert(actual, expected, 'timex')
```

Comment:
should we use single_assert method to the others asserts_model_resolutions_xxx methods

Reply:
Nice catch! Now all assertions are made with `single_assert`